addons/web/tests/test_load_menus.py

```diff
@@ -0,0 +1,48 @@
+from odoo.tests.common import HttpCase
+
+class LoadMenusTests(HttpCase):
+
+    def setUp(self):
+        super().setUp()
+        self.menu = self.env["ir.ui.menu"].create({
+            "name": "test_menu",
+            "parent_id": False,
+        })
+
+        def search(*args, **kwargs):
+            return self.menu
+
+        self.patch(type(self.env["ir.ui.menu"]), "search", search)
+        self.authenticate("admin", "admin")
+
+    def test_load_menus(self):
```

Comment:
Issue with tests without docstring or assert messages is that if it failed, you don't really know what is tested